controllers/user.controller.js

```diff
@@ -0,0 +1,63 @@
+// controllers/userController.mjs
+import pool from '../dbConfig.mjs';
+
+export const createUser = async (req, res) => {
+  try {
+    const { username, age, email } = req.body;
+    const query = `
+      INSERT INTO users (username, age, email)
+      VALUES ($1, $2, $3)
+      RETURNING *;
+    `;
+    const values = [username, age, email];
+    const result = await pool.query(query, values);
+    res.json(result.rows[0]);
```

Comment:
It's advised to include a status code even if it's successful, with an appropriate response body. Let's work with uniformity here.
```suggestion
    res.status(<appropriate status code>).json({
    "message":"User created successfully",
    "data": result.rows[0]});
```
We can apply the same structure to all successful responses.

`TODO`: Look for appropriate status code to be used when a resource is successfully created on the backend.